src/compiler/visitor.ts

```diff
@@ -46,54 +46,54 @@ namespace ts {
      *       supplant the existing `forEachChild` implementation if performance is not
      *       significantly impacted.
      */
-    const nodeEdgeTraversalMap = createMap<NodeTraversalPath>({
-        [SyntaxKind.QualifiedName]: [
+    const nodeEdgeTraversalMap = createMapFromPairs<NodeTraversalPath>(

```

Comment:
Eventually I plan to get rid of this. I found it was too expensive for most nodes. Until that time, just make this into a sparse array.